src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerInEstablishmentService.kt

```diff
@@ -0,0 +1,197 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.services
+
+import com.google.gson.Gson
+import com.microsoft.applicationinsights.TelemetryClient
+import org.elasticsearch.action.search.SearchRequest
+import org.elasticsearch.action.search.SearchResponse
+import org.elasticsearch.common.unit.TimeValue
+import org.elasticsearch.index.query.BoolQueryBuilder
+import org.elasticsearch.index.query.MultiMatchQueryBuilder
+import org.elasticsearch.index.query.Operator
+import org.elasticsearch.index.query.QueryBuilder
+import org.elasticsearch.index.query.QueryBuilders
+import org.elasticsearch.search.builder.SearchSourceBuilder
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.PageImpl
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.prisonersearch.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.security.AuthenticationHolder
+import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.PaginationRequest
+import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.PrisonerInEstablishmentRequest
+import java.util.concurrent.TimeUnit
+
+@Service
+class PrisonerInEstablishmentService(
+  private val elasticsearchClient: SearchClient,
+  private val indexStatusService: IndexStatusService,
+  private val gson: Gson,
+  private val telemetryClient: TelemetryClient,
+  private val authenticationHolder: AuthenticationHolder,
+  @Value("\${search.keyword.max-results}") private val maxSearchResults: Int = 200,
+  @Value("\${search.keyword.timeout-seconds}") private val searchTimeoutSeconds: Long = 10L,
+) {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  fun search(prisonId: String, prisonerSearchRequest: PrisonerInEstablishmentRequest): Page<Prisoner> {
+    log.info("Received keyword request ${gson.toJson(prisonerSearchRequest)}")
```

Comment:
does this need to be logged as `json`, why not just use `toString` on the request to log normally?